script.js

```diff
@@ -0,0 +1,149 @@
+const headerNavbar = document.getElementById('navbarSupportedContent');
+const resultBlock = document.getElementById('results');
+const pages = document.querySelector('.pagination');
+const searchButton = document.getElementById('search');
+const searchInput = document.querySelector('.form-control');
+const loader = document.querySelector('.loader');
+let currentPage = 1;
+const maxInfoLinesCount = 5;
+const API_URL = 'https://swapi.dev/api/';
+let searchMode = false;
+const LocalStorageKeys = {
+    category: 'category'
+}
+const selectedCategory = localStorage.getItem(LocalStorageKeys.category) || CATEGORIES.people;
+
+document.addEventListener("DOMContentLoaded", () => {
+    headerNavbar.querySelector(`#${selectedCategory}`).checked = true;
+    getData(selectedCategory).then((data) => {
+        renderCards(data, selectedCategory);
+        addPagination(data.count);
+    })
+});
+
+
+function getData(path) {
+    resultBlock.innerHTML = null;
+    loader.classList.remove(MODIFICATORS.hide);
+    return fetch(`${API_URL}${path}`).then((res) => res.json());
+}
+
+function getSelectedCategory() {
+    return [...headerNavbar.querySelectorAll('input')].find(item => item.checked).getAttribute('id');
+}
+
+pages.addEventListener('click', ({target}) => {
+    const pageItem = target.closest('.page-item');
+    if (!pageItem.classList.contains('.active')) {
+        pages.querySelectorAll('li').forEach(item => item.classList.remove('active'));
+        pageItem.classList.add('active');
+        const selectedCategory = getSelectedCategory();
+        getData(`${selectedCategory}/?page=${pageItem.getAttribute('data-index')}`)
+            .then((data) => renderCards(data, selectedCategory));
+    }
+})
+
+
+resultBlock.addEventListener('click', (event) => {
+    if (event.target.closest('.btn') && event.isTrusted) {
```

Comment:
You can use destructuring of ```event``` for ```target``` and ```isTrusted``` values.

Reply:
```suggestion
resultBlock.addEventListener('click', ({target, isTrusted}) => {...}
```
Fixed